Resources/Prototypes/Entities/Mobs/Player/humanoid.yml

```
@@ -14,6 +14,14 @@
     - type: MindShield
     - type: AntagImmune
 
+- type: randomHumanoidSettings
+  id: EventHumanoidCentcomm
+  parent: EventHumanoidMindShielded
+  components:
+    - type: AutoImplant
+      implants:
+      - DeathRattleImplantCentcomm
```

```suggestion
  components:
  - type: AutoImplant
    implants:
    - DeathRattleImplantCentcomm
```
fix indentation
(yes, I know it's wrong above as well)